### ISSUE TITLE
`default` literal of tuple types does not trigger nullability check

### ISSUE THREAD

**Issue by PopSlime** [dotnet/roslyn#77693]
**Version Used**: Visual Studio Community 2022 Version 17.14.0 Preview 2.0

**Steps to Reproduce**:

Consider the following code:

```cs
(string, string) tuple = default;
```

**Diagnostic Id**:

CS8619: Nullability of reference types in value of type '(string?, string?)' doesn't match target type '(string, string)'.

**Expected Behavior**:

The diagnostic above is generated.

**Actual Behavior**:

The diagnostic above is not generated.

**Comment by huoyaoyuan** [comment]
The compiler is following the language specification here. See https://github.com/dotnet/csharplang/discussions/8718 .

The default value of tuple types aren't analyzed element-wise, like any other struct.
